packages/react-components/src/components/VideoGallery.tsx

```diff
@@ -176,8 +179,10 @@ export const VideoGallery = (props: VideoGalleryProps): JSX.Element => {
             onCreateRemoteStreamView={onCreateRemoteStreamView}
             onDisposeRemoteStreamView={onDisposeRemoteStreamView}
             isAvailable={remoteVideoStream?.isAvailable}
+            renderElement={
+              !maxRemoteVideoStreams || index < maxRemoteVideoStreams ? remoteVideoStream?.renderElement : undefined
```

Comment:
From Alex's comment the reason this is being implemented is due to bandwidth and webrtc issues, since this change is purely UI being changing I want to confirm this change does reduce the bandwidth/webrtc problems (do we only consume the bandwidth if we use the render element or is stateful still creating it?)

Reply:
Good point. From what I remember, we only consume bandwidth when then renderElement is actually rendered in the DOM. Can someone confirm this?
@alkwa-msft @prprabhu-ms

Reply:
when we startRender that is when we start using more of the bandwidth

Reply:
From your comment alex sounds like we also need to ensure onCreateRemoteStreamView isn't being called (which, with only very lightly skimming this file, looks like its being called
![image](https://user-images.githubusercontent.com/2684369/123473737-d7b5b380-d5ad-11eb-970b-7600059d0b76.png)
)

Reply:
yes and since we're using the condition of the maxRemoteStreams as part of the renderElement boolean I think we're good here.